llama.cpp

```diff
@@ -2248,3 +2249,122 @@ const char * llama_print_system_info(void) {
 std::vector<std::pair<std::string, struct ggml_tensor *>>& llama_internal_get_tensor_map(struct llama_context * ctx) {
     return ctx->model.tensors_by_name;
 }
+
+// Returns the size of the state
+size_t llama_get_state_size(struct llama_context * ctx) {
+    const size_t s_bool = sizeof(int32_t);
```

Comment:
`s_bool` is unused - is this expected?

Reply:
Oh, I missed that during cleanup of unused stuff.  
At one time during implementation I was saving the bool flags from llama_context, but removed it because it didnt make much sense.